tests/grids/test_grid.py

```diff
@@ -132,6 +132,218 @@ def test_cell_faces_as_dense(g: pp.Grid):
     assert np.allclose(reconstructed.toarray(), cf_sparse.toarray())
 
 
+class TestDivergenceTrace:
+
+    def create_grids(self) -> list[pp.Grid]:
+        """Create grids used for testing the divergence and trace operators.
+
+        The grids are:
+            1. 2d Cartesian grid with 2x2 cells.
+            2. 3d Cartesian grid with 2x2x2 cells.
+            3. 2d structured triangle grid with 2x2 cells.
+            4. 3d structured tetrahedral grid with 2x2x2 cells.
+            5  A 1d grid with two cells, and with the middle node split.
+
+        Returns:
+            A list of grids.
+
+        """
+        g1 = pp.CartGrid([2, 2], [1, 1])
+        g2 = pp.CartGrid([2, 2, 2], [1, 1, 1])
+        g3 = pp.StructuredTriangleGrid([2, 2], [1, 1])
+        g4 = pp.StructuredTetrahedralGrid([2, 2, 2], [1, 1, 1])
```

Comment:
Do you think we should add an unstructured as well for good measure, in case structure of these grids masks something?

Reply:
I thought about it, but don't see how this will add true information; in this setting, the structured simplex grids are somehow unstructured.